src/screens/CreatePost/index.js

```diff
@@ -309,20 +312,13 @@ const CreatePost = () => {
 
   const uploadPhotoImage = async (pathImg) => {
     try {
-      const compressionResult = await Image.compress(pathImg, {
-        compressionMethod: 'auto'
-      });
-
+      const compressionResult = await ImageCompressionUtils.compress(pathImg);
       const newArr = {
         id: mediaStorage.length,
         data: compressionResult
       };
 
-      const asset = new FormData();
-      asset.append('photo', composeImageMeta(compressionResult));
-
-      const responseUpload = await uploadPhoto(asset);
-
+      const responseUpload = await ImageUtils.uploadImage(pathImg);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The image compression and upload logic has been updated. The new logic uses `ImageCompressionUtils.compress` for compression and `ImageUtils.uploadImage` for uploading. Ensure that these functions are implemented correctly in their respective modules and return the expected results.
```diff
-      const compressionResult = await Image.compress(pathImg, {
-        compressionMethod: 'auto'
-      });
-
-      const asset = new FormData();
-      asset.append('photo', composeImageMeta(compressionResult));
-
-      const responseUpload = await uploadPhoto(asset);
+      const compressionResult = await ImageCompressionUtils.compress(pathImg);
+      const newArr = {
+        id: mediaStorage.length,
+        data: compressionResult
+      };
+      const responseUpload = await ImageUtils.uploadImage(pathImg);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->